lib/IRremoteESP8266/src/IRutils.cpp

```diff
@@ -103,7 +103,7 @@ decode_type_t strToDecodeType(const char * const str) {
 /// @param[in] isRepeat A flag indicating if it is a repeat message.
 /// @return A String containing the protocol name. kUnknownStr if no match.
 String typeToString(const decode_type_t protocol, const bool isRepeat) {
-  String result = "";
+  String result = F("");
```

Comment:
`EMPTY_STRING`

Reply:
That is in a library, so what should I #include to have access to `EMPTY_STRING`? 🤔

Reply:
Ah sorry, in the library you cannot use it.
The ESP8266 implementation of WString class does have a similar static const object, but the ESP32 version doesn't, so you cannot use it there.

Also for edits in libraries, please keep in mind the library should also be maintainable with future releases of the maintainer of that library.
That specific library is being maintained quite frequently, so rather not make too specific ESPEasy modifications to it.